blocks/library/embed/index.js

```diff
@@ -1,7 +1,9 @@
 /**
  * WordPress dependencies
  */
-import { Button, Placeholder } from 'components';
+import Button from 'components/button';
+import Placeholder from 'components/placeholder';
+import HtmlEmbed from 'components/html-embed';
```

Comment:
I think this branch is already rebased, because in the fix I just pushed, the spinner is imported using that syntax. However I got a Dashicons error when I tried using `import { Button, Placeholder, HtmlEmbed, Spinner } from 'components';` as the syntax.